deepchecks/core/check.py

```diff
@@ -206,6 +212,79 @@ def _display_to_json(self) -> List[Tuple[str, str]]:
         matplotlib.use(old_backend)
         return displays
 
+    def to_wandb(self, wandb_init: bool = True, **kwargs: Any):
+        """Export check result to wandb.
+
+        Parameters
+        ----------
+        wandb_init : bool , default: True
+            if to initiate a new wandb run
+        kwargs: Keyword arguments to pass to wandb.init - relevent if wandb_init is True.
+        """
+        assert wandb, 'Missing wandb dependency, please install wandb'
+        check_metadata = self._get_metadata()
+        if wandb_init:
+            kwargs['project'] = kwargs.get('project', check_metadata['name'])
+            kwargs['project'] = kwargs.get('config', check_metadata)
+            wandb.init(**kwargs)
+            section_suffix = ''
+        else:
+            section_suffix = check_metadata['name'] + '/'
+        if self.conditions_results:
+            cond_df = get_conditions_table([self], icon_html=False)
+            cond_table = wandb.Table(dataframe=cond_df.data, allow_mixed_types=True)
+            wandb.log({f'{section_suffix}conditions_table': cond_table}, commit=False)
+        if isinstance(self.value, pd.DataFrame):
+            value = self.value.to_json()
+        elif isinstance(self.value, Styler):
+            value = self.value.data.to_json()
+        elif isinstance(self.value, np.ndarray):
+            value = self.value.tolist()
+        else:
+            value = jsonpickle.dumps(self.value)
```

Comment:
Isn't it just the same as the to_json method? maybe we should reuse the code from there..?

Reply:
not all the lines